tests/test_features/test_manifold.py

```diff
@@ -117,46 +119,80 @@ def test_manifold_fit(self, mock_fit_transform):
         manifold = Manifold(target="auto")
 
         assert manifold.fit(X, y) is manifold, "fit did not return self"
-        mock_fit_transform.assert_called_once()
```

Comment:
If this is being removed, is there any reason to mock the `fit_transform` method?

Reply:
My bad!